bc_obps/service/contact_service.py

```diff
@@ -37,6 +41,21 @@ def get_if_authorized(cls, user_guid: UUID, contact_id: int) -> Optional[Contact
             raise Exception(UNAUTHORIZED_MESSAGE)
         return contact
 
+    @classmethod
+    def get_with_places_assigned(cls, user_guid: UUID, contact_id: int) -> Optional[ContactWithPlacesAssigned]:
+        contact = cls.get_if_authorized(user_guid, contact_id)
+        places_assigned = []
+        if contact:
+            role_name = contact.business_role.role_name
+            for operation in contact.operations_contacts.all():
+                places_assigned.append(f"{role_name} - {operation.name}")
+            # Return the Contact plus places_assigned
+            result = cast(ContactWithPlacesAssigned, contact)
+            if len(places_assigned) > 0:
```

Comment:
We don't need this check, empty arrays in Python are falsy.

Reply:
Thanks, I had this come up in another PR too, that'll save me some time!